### DISCUSSION TITLE
Column width

### DISCUSSION THREAD

**Discussion by justich** [Budibase/budibase#1861] [Help] [answered]
Just having a play. Love what you are doing thus far.

I'm trying to add an image field to a table.

![image](https://user-images.githubusercontent.com/22791072/123625718-f3b49180-d807-11eb-8161-e1e36a11a4d5.png)

How can I expand the row to fit the Heading / Image fields? 

Can custom fields on a table be added to the Table header row and have sort actions?

**Comment by mjashanks** [comment, marked as answer]
Hey @justich . I've had a play around with this, and can't find a solution for you. Our main purpose for this feature (adding children to table) is to allow you to add Buttons and Links.

Could you use a `Repeater` instead of a table? A repeater will repeat its content for every row from the datasource provided.

**Comment by justich** [comment]
Great. Thanks for your speedy response. Just out of interest is there another way to add an image to a table without using a repeater?

**Reply by mjashanks** [reply]
If you are using the Budibase internal database, you can create a column of type `Attachment`. This will allow you to attach images to a record. 

This will be displayed as a thumbnail in the table component.

Or, using your method above, you could set the `Max Width` of the image itself (via the design panel) to make it fit.
